### PR TITLE
Better visibility for benchmark improvements/regressions

### PR DESCRIPTION
**Pull request by nerfZael**
Can update benchmarks.json by adding `-s` at the beginning:
`python benchmarks.py -s ./autotx/tests/agents/token 5`
Alternatively, to not update it run:
`python benchmarks.py ./autotx/tests/agents/token 5`

**Proposed PR process:**
1. Open PR
2. Run benchmarks
3. See improvements and/or regressions
4. Once PR is done, run final benchmarks (can be just a sub dir) with `-s` and commit `benchmarks.json`

### CODE REVIEW

**Comment by nerfZael** [comment]
/workflows/benchmarks agents/token 5

**Comment by nerfZael** [comment]
/workflows/benchmarks agents/token

File: benchmarks.py [hunk 1]
```diff
@@ -63,13 +64,7 @@ def run_test(test_name, iterations, avg_time_across_tests, completed_tests, rema
...
     return pass_count, fail_count, run_times
 
-def clear_lines(n=1):
-    """Clears a specified number of lines in the terminal."""
-    for _ in range(n):
-        sys.stdout.write('\x1b[1A')  # Move the cursor up by one line
-        sys.stdout.write('\x1b[2K')  # Clear the current line
-
-def print_summary_table(test_path, iterations, tests_results, total_run_time, output_dir):
+def print_summary_table(test_path, iterations, tests_results, total_run_time, output_dir, total_benchmarks):
```
Comment (cbrzn) [inline benchmarks.py:67]: super nit: if we could add types here would be awesome hehe

**Review by cbrzn — APPROVED** [review]

File: benchmarks.py (hunk 1 above)
Comment (nerfZael) [reply benchmarks.py]: done

**Comment by nerfZael** [comment]
/workflows/benchmarks agents/token/send/test_send.py::test_auto_tx_send_native 1

**Comment by nerfZael** [comment]
/workflows/benchmarks agents/token/test_swap_and_send.py::test_auto_tx_swap_and_send_complex 3

**Comment by nerfZael** [comment]
/workflows/benchmarks agents/token

**Pull request merged**